Dockerfile.crossbuild

```diff
@@ -11,9 +12,9 @@ ENV KAFKA_VERSION=$kafka_version \
     PATH=${PATH}:${KAFKA_HOME}/bin
 
 COPY download-kafka.sh start-kafka.sh broker-list.sh create-topics.sh /tmp/
-
-RUN apk add --update unzip wget curl docker jq coreutils \
- && chmod a+x /tmp/*.sh \
+RUN mkdir -p /opt
+RUN apk add --update unzip wget curl docker jq coreutils bash \
+ && chmod a+x /tmp/*.sh && sync \
```

Comment:
Where does this `sync` come from?

Reply:
Without this `sync` - I was seeing this issue :  https://github.com/wurstmeister/kafka-docker/issues/138
So i forward ported the fix that's presently here: https://github.com/wurstmeister/kafka-docker/blob/0.10.0/Dockerfile#L12